etc/scipipe/build_matrix.yaml

```diff
@@ -16,13 +16,15 @@ template:
     - &el7-conda
       <<: *platform_defaults
       image: docker.io/lsstdm/scipipe-base:7
-      label: centos-7-conda
+      display_name: linux-7-x86
+      label: docker
       compiler: conda-system
       python: '3'
     - &el8-conda
       <<: *platform_defaults
-      image: docker.io/lsstdm/scipipe-base:8
-      label: centos-8-conda
+      image: ghcr.io/lsst-dm/docker-scipipe:9-latest
```

Comment:
Why is this under `el8-conda`?  Wouldn't `linux-9-arm` be better?  (If it is running on Alma9, which I think it is from that image.)